Detectors/TRD/workflow/include/TRDWorkflow/TRDEventDisplayFeedSpec.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <gsl/span>
+
+#include "TRDBase/Geometry.h"
+
+#include "Framework/DataProcessorSpec.h"
+#include "Framework/Task.h"
+
+#include "DataFormatsGlobalTracking/RecoContainer.h"
+#include "DataFormatsTRD/TrackTRD.h"
+#include "DataFormatsTRD/TriggerRecord.h"
+#include "DataFormatsTRD/Digit.h"
+
+#include "TRDBase/TrackletTransformer.h"
+
+namespace o2
+{
+namespace trd
+{
+
+class TRDEventDisplayFeedSpec : public o2::framework::Task
+{
+ public:
+  TRDEventDisplayFeedSpec(){};
+  ~TRDEventDisplayFeedSpec() override = default;
+  void init(o2::framework::InitContext& ic) override;
+  void run(o2::framework::ProcessingContext& pc) override;
+  void printTracks(gsl::span<const TrackTRD> tracks, gsl::span<const Tracklet64> tracklets, gsl::span<const TrackTriggerRecord> trackTrigRecs, int iEvent);
+  void printTracklets(gsl::span<const Tracklet64> tracklets, int iEvent);
+  void writeDigits(gsl::span<const Digit> digits, int iEvent);
+
+ private:
+  o2::trd::Geometry* mGeo;
+  TrackletTransformer mTransformer;
+  float mBz;
+
+  gsl::span<const TriggerRecord> mTrigRecs;
+  std::map<std::string, std::string> mTrackletMap;
+  std::ofstream* summary = new std::ofstream();
+
+  bool mUsedDetectors[540];
```

Comment:
Prefer `std::array<bool, constants::MAXCHAMBER> mUsedDetectors{};`. You will need to include `DataFormatsTRD/Constants.h`.